fe/fe-core/src/main/java/org/apache/doris/resource/resourcegroup/ResourceGroupMgr.java

```diff
@@ -149,6 +151,30 @@ public void createResourceGroup(CreateResourceGroupStmt stmt) throws DdlExceptio
         LOG.info("Create resource group success: {}", resourceGroup);
     }
 
+    public void dropResourceGroup(DropResourceGroupStmt stmt) throws DdlException {
+        if (!Config.enable_resource_group) {
+            throw new DdlException("unsupported feature now, coming soon.");
+        }
+
+        String resourceGroupName = stmt.getResourceGroupName();
```

Comment:
The default resource group should not be allowed to be dropped.

Reply:
Done